WalletWasabi/Mono/CommandSet.cs

```diff
@@ -450,10 +450,6 @@ public async Task<int> RunAsync(IEnumerable<string> arguments)
 				{
 					return await Help.InvokeAsync(extra);
 				}
-				if (arguments.All(x => !x.Contains("version")))
-				{
-					Out.WriteLine(Options.MessageLocalizer($"Use `{Suite} help` for usage."));
-				}
```

Comment:
@lontivero Would it be safe to merge only this change without the rest of the modifications? (I'd delay the rest for later, maybe to after feature freeze.)

Reply:
I think it is not safe to merge only those lines because you added them to prevent to launch the GUI when the `--version` is among the arguments.